test/module/Entra/Set-EntraDirSyncFeature.Tests.ps1

```diff
@@ -0,0 +1,82 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+BeforeAll {  
+    if((Get-Module -Name Microsoft.Graph.Entra) -eq $null){
+        Import-Module Microsoft.Graph.Entra       
+    }
+    Import-Module (Join-Path $psscriptroot "..\Common-Functions.ps1") -Force
+    $scriptblock = {
+        return @(
+            [PSCustomObject]@{
+                "Configuration" = @{ AlertThreshold =500 ; SynchronizationPreventionType = "enabledForCount"}
+                "Id" = "00aa00aa-bb11-cc22-dd33-44ee44ee44ee"
+            }        
+        )
+        }
+    Mock -CommandName Get-MgDirectoryOnPremiseSynchronization -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+    
+    Mock -CommandName Update-MgDirectoryOnPremiseSynchronization -MockWith {} -ModuleName Microsoft.Graph.Entra
+}
+Describe "Set-EntraDirSyncFeature" {
+    Context "Test for Set-EntraDirSyncFeature" {
+        It "Should sets identity synchronization features for a tenant." {
+            $result =  Set-EntraDirSyncFeature -Feature "BypassDirSyncOverrides" -Enable $false -TenantId "00aa00aa-bb11-cc22-dd33-44ee44ee44ee" -Force
```

Comment:
Docs have `Enabled` as the parameter.
![image](https://github.com/user-attachments/assets/762b96ca-7f8b-4f4a-9377-abcc1e26dd6b)
Ps1 file has `Enabled` https://github.com/microsoftgraph/entra-powershell/blob/2cfcd9a542909d717731c08f9ecd5afff02869e8/module/Entra/AdditionalFunctions/Set-EntraDirSyncFeature.ps1#L9

These tests have `Enable`. Should be fixed.